etcd/etcd-cluster-runtime-reconfiguration-on-coreos.md

```diff
@@ -0,0 +1,187 @@
+# etcd Cluster Runtime Reconfiguration on CoreOS
+
```

Comment:
We still need a really high level overview here. Probably the same comment from https://github.com/coreos/docs/pull/734/files#r51036485 or similar.